packages/core/src/amazonq/webview/ui/diffTree/actions.ts

```diff
@@ -10,10 +10,16 @@ import { DiffTreeFileInfo } from './types'
 export function getDetails(filePaths: DiffTreeFileInfo[]): Record<string, TreeNodeDetails> {
     const details: Record<string, TreeNodeDetails> = {}
     for (const filePath of filePaths) {
-        if (filePath.rejected) {
+        if (filePath.changeApplied) {
+            details[filePath.relativePath] = {
+                status: 'success',
+                label: 'Change accepted',
+                icon: MynahIcons.OK,
+            }
+        } else if (filePath.rejected) {
             details[filePath.relativePath] = {
                 status: 'error',
-                label: 'File rejected',
+                label: 'Change rejected',
```

Comment:
Can we import these label strings from i18n file?

Reply:
i18n does not work In this particular package `core/src/amazonq`. Updated i18n everywhere else.
Moved this to `uiComponentsTexts` instead.

Reply:
Got it. Let's keep this way then for these